tests/Unit/Evolution/Systems/GeneralizedHarmonic/Test_Characteristics.cpp

```diff
@@ -29,7 +29,7 @@
 #include "PointwiseFunctions/GeneralRelativity/ComputeSpacetimeQuantities.hpp"
 #include "PointwiseFunctions/GeneralRelativity/IndexManipulation.hpp"
 #include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
-#include "Utilities/ConstantExpressions.hpp"
+#include "Utilities/ConstantExpressions.hpp"  //IWYU pragma: keep
```

Comment:
Why are you "pragma: keep"ing this?  I don't think you use it.

Reply:
ah, I thought there was a `square` in there, but there wasn't. removed the `include`.